tests/Tests.FeatureManagement/FeatureManagement.cs

```diff
@@ -84,7 +85,7 @@ public async Task Integrates()
 
                     services.AddMvcCore(o =>
                     {
-                        o.EnableEndpointRouting = false;
```

Comment:
How does enabling or disabling this option affect the behavior of our test project?

Reply:
The tests were originally written to use the ASP.NET Core 2.1 test host, so that use the original routing strategy instead of endpoint routing which was introduced in ASP.NET Core 3.1. 3.1 had a breaking change to where you can't call `UseMvc` anymore without explicitly setting `EnableEndpointRouting` to false. So any test that calls `UseMvc` needs to have this.